src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,42 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        try (BufferedReader reader = new BufferedReader(new FileReader(new File(fromFileName)));
+                 BufferedWriter writer = new BufferedWriter(new FileWriter(new File(toFileName)))) {
+
+            int supplyTotal = 0;
+            int buyTotal = 0;
+            String line;
+            while ((line = reader.readLine()) != null) {
+                String[] parts = line.split(",");
```

Comment:
for this comma symbol as well

Reply:
add a constant with comma values and use it here